.travis.yml

```diff
@@ -1,16 +1,41 @@
-language: generic
+---
+
+language: node_js
+
+node_js:
+  - lts/*
+
+sudo: required
 
 services:
- - docker
+  - docker
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - libstdc++-5-dev
 
-env:
- - DEPENDENCY_PACKAGES="appstream desktop-file-utils libclutter-gst-3.0-dev libclutter-gtk-1.0-dev libgranite-dev libgtk-3-dev valac libcanberra-dev meson"
+cache:
+  directories:
+    - /tmp/liftoff
+
+matrix:
+  include:
+    - env: DIST=juno
+
+before_install:
+  - docker pull ubuntu:18.04
 
 install:
- - docker pull elementary/docker:loki
- - docker run -v "$PWD":/tmp/build-dir elementary/docker:loki /bin/sh -c "apt-get update && apt-get -y install $DEPENDENCY_PACKAGES && cd /tmp/build-dir && meson build && cd build && ninja test"
- - docker pull elementary/docker:loki-unstable
- - docker run -v "$PWD":/tmp/build-dir elementary/docker:loki-unstable /bin/sh -c "apt-get update && apt-get -y install $DEPENDENCY_PACKAGES && cd /tmp/build-dir && rm -rf build && meson build && cd build && ninja test"
+  - npm install @elementaryos/houston
```

Comment:
We should start changing this to `npm install @elementaryos/houston@1` to prevent breaking changes breaking CI